### PR TITLE
Feature auto ingestion

### PR DESCRIPTION
**Pull request by flowstef**
Groundwork for ingesting data from command line as opposed to running the ingestion notebook.
Triggered via new make target (for dev environment) or directly using docker compose.

This is the prerequisite for adding data for end to end cypress tests.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -250,6 +251,10 @@ run: clear
 		"cd ./src; \
 		uvicorn $(PACKAGE_NAME).main:app $(RELOAD) --host 0.0.0.0 --port $(SERVER_PORT_CONTAINER);"
 
+ingest:
+	$(info Running ingestion script to populate database...)
+	export JUPYTER_PORT=$(JUPYTER_PORT); export CONTAINER_USER=$(CONTAINER_USER); export UID=$(UID); export GID=$(GID); docker-compose -p $(APP_NAME) --env-file ./.env run --entrypoint="bash -c \"pip install httpx pandas; cd /home/$(CONTAINER_USER); jupyter nbconvert --to script PopulateDatabase.ipynb; cat PopulateDatabase.py | grep -v get_ipython > run.py; python run.py; rm PopulateDatabase.py run.py\"" jupyter
```
Comment (greenape) [inline Makefile:256]: Why convert to a script and run instead of just running using nbconvert?

Comment (greenape) [inline Makefile:256]: Why's this going via the entrypoint rather than being run as a command?

File: src/jupyter/PopulateDatabase.ipynb [hunk 2]
```diff
@@ -299,27 +302,27 @@
...
       "            \"label_fr\": \"Changement de r\\u00e9sidence\",\n",
       "            \"description_fr\": \"Les indicateurs relatifs au changement de r\\u00e9sidence d\\u00e9crivent les variations (mensuelles) \\u00e0 long terme du nombre de personnes qui changent de lieu de r\\u00e9sidence entre des paires de zones.\"\n",
       "        },\n",
       "        {\n",
-      "            \"category_id\": \"visitors\",\n",
+      "            \"category_id\": \"presence\",\n",
       "            \"type\": \"single_location\",\n",
       "            \"order\": 3,\n",
-      "            \"flowgeek_url\": \"https://www.flowgeek.org/from-data-to-insights/calculating-mobility-indicators/presence-indicators/\",\n",
+      "            \"flowgeek_url\": \"https://www.flowgeek.org/methods/calculating-mobility-indicators/presence-indicators/\",\n",
```
Comment (greenape) [inline src/jupyter/PopulateDatabase.ipynb:315]: This is a cheeky unrelated fix, right?

File: src/jupyter/PopulateDatabase.ipynb [hunk 3]
```diff
@@ -31,11 +31,11 @@
    "id": "92716416-e32a-4944-9af2-23b9ac53d576",
```
Comment (greenape) [inline src/jupyter/PopulateDatabase.ipynb:31]: <pre class="ql-syntax" spellcheck="false">Line #144.    async def doit():
</pre><p>This bit is necessary because it is being run as a script not within the nb which has magic for handling async?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Flowminder/flowkit-ui-backend/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

File: Makefile (hunk 1 above)
Comment (flowstef) [reply Makefile:256]: I didn't know you could do that!

Comment (flowstef) [reply Makefile:256]: Because the entrypoint currently starts a jupyter notebook server with all sorts of quality of life improvements (e.g. dark mode) which takes time. We don't need that at all for a command line script, I just wanted to void the original entrypoint hence overwriting it.

File: src/jupyter/PopulateDatabase.ipynb (hunk 2 above)
Comment (flowstef) [reply src/jupyter/PopulateDatabase.ipynb:315]: Yes. As I was debugging the ingestion (because some indicators were missing after the ingestion had finished) I found inconsistent naming so I fixed it all in one go.
Apologies!

File: Makefile (hunk 1 above)
Comment (greenape) [reply Makefile:256]: [jupyter nbconvert --to notebook --exec PopulateDatabase.ipynb](https://nbconvert.readthedocs.io/en/latest/execute_api.html#executing-notebooks-from-the-command-line) oughta do it.

File: src/jupyter/PopulateDatabase.ipynb (hunk 3 above)
Comment (flowstef) [reply src/jupyter/PopulateDatabase.ipynb:31]: Yes exactly. While running in a nb there's a loop running but on command line there isn't.
I wanted both to work.

File: Makefile (hunk 1 above)
Comment (greenape) [reply Makefile:256]: Looks like the _command_ does, but the entry point isn't specified so it'll otherwise be the image's defaul, which is https://github.com/jupyter/docker-stacks/blob/main/docker-stacks-foundation/start.sh

Comment (flowstef) [reply Makefile:256]: No, doesn't work:
```
[NbConvertApp] WARNING | Unrecognized alias: 'exec', it will have no effect.
```
but after reading tfm myself I found `--execute` to be working.
Although that gives you no output so I think I prefer the version I stole from [a forum post](https://discourse.jupyter.org/t/how-to-run-a-notebook-using-command-line/3475/12).

Comment (greenape) [reply Makefile:256]: Aye, you'd get the output back in the notebook. Wanting to have it via stdout suffices as a reason to go this way for me here.

Comment (flowstef) [reply Makefile:256]: You're right. Changing that.

**Review by greenape — APPROVED** [review]
Good for me.
